testsuite/tests/typing-layouts-untagged-immediate/untagged_ints.ml

```diff
@@ -1,17 +1,17 @@
 (* TEST
- include stdlib_beta;
+ include stdlib_stable;
  flambda2;
  {
-   flags = "-extension layouts_alpha -extension small_numbers_beta";
+   flags = "-extension-universe alpha";
    native;
  }{
-   flags = "-extension layouts_alpha -extension small_numbers_beta";
+   flags = "-extension-universe alpha";
    bytecode;
  }{
-   flags = "-extension layouts_beta -extension small_numbers_beta";
+   flags = "-extension-universe beta";
    native;
  }{
-   flags = "-extension layouts_beta -extension small_numbers_beta";
+   flags = "-extension-universe beta";
```

Comment:
I think I expect us to be testing `stable` too?  And if `alpha` has the same output as `stable` and `beta`, we can delete the `untagged_ints_alpha.ml` file and combine it with this one.